tests/timeseries/_formula_engine/test_formula_composition.py

```diff
@@ -171,16 +171,70 @@ async def test_formula_composition_missing_bat(self, mocker: MockerFixture) -> N
         assert count == 10
 
     async def test_formula_composition_min_max(self, mocker: MockerFixture) -> None:
-        """Test the composition of formulas with min/max values."""
+        """Test the composition of formulas with the min and max"""
         mockgrid = MockMicrogrid(grid_meter=True)
+        mockgrid.add_chps(1)
         await mockgrid.start(mocker)
 
         logical_meter = microgrid.logical_meter()
-        engine_min = logical_meter.grid_power._min(  # pylint: disable=protected-access
+        engine_min = logical_meter.grid_power.min(  # pylint: disable=protected-access
```

Comment:
do you know why this still needs disable protected-access?

Reply:
I'm pretty sure it's not needed. Good catch!